imap_processing/cdf/config/imap_ialirt_global_cdf_attrs.yaml

```
@@ -0,0 +1,18 @@
+instrument_base: &instrument_base
+  Descriptor: IALiRT>IMAP IALiRT (IMAP Active Link in Real Time)
```

Do you need to include an extra hyphen in the name?
```suggestion
  Descriptor: IALiRT>IMAP I-ALiRT (IMAP Active Link in Real Time)
```